default.nix

```diff
@@ -1,30 +1,26 @@
 { pkgs ? (import <nixpkgs> {}), coq-version ? "master" }:
 
 let
- coq =
-   if coq-version == "8.8" then pkgs.coq_8_8
-   else import (fetchTarball "https://github.com/coq/coq/tarball/${coq-version}") {};
-
- ssreflect =
-   if coq-version == "8.8" then pkgs.coqPackages_8_8.ssreflect
-   else import ./ssreflect.nix { inherit pkgs coq; };
+  coqPackages =
+    if coq-version == "8.8" then pkgs.coqPackages_8_8 else
+    if coq-version == "8.9" then pkgs.coqPackages_8_9 else
+    pkgs.mkCoqPackages
+      (import (fetchTarball "https://github.com/coq/coq/tarball/${coq-version}") {
+        buildIde = false;
+        buildDoc = false;
+        doInstallCheck = false;
+        coq-version = "8.9";
```

Comment:
Overriding these parameters is actually not a good idea as this won't allow benefiting from the binary cache provided through Cachix.

Reply:
That's one more reason BTW why we might want to split these various things (Coq core, CoqIDE, the documentation) in several packages.

Reply:
@vbgl NixOS/nixpkgs#49780 should fix this issue.